finagle-http/src/main/scala/com/twitter/finagle/http/codec/ConnectionManager.scala

```diff
@@ -11,7 +11,7 @@ import com.twitter.util.{Future, Promise}
 private[finagle] class ConnectionManager {
   private[this] var isKeepAlive = false
   private[this] var isIdle = true
-  private[this] var activeStreams = 0
+  private[this] var activeStreams, requestsObserved, responsesObserved = 0
```

Comment:
FWIF, I'm voting for a single variable here (`activeRequests` or `pendingResponses`).